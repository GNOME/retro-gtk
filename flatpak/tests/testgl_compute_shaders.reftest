[Retro Reftest]
Path=/testgl_compute_shaders
Core=/app/lib/libretro/testgl_compute_shaders_libretro.so

[Options]
boxes_resolution=320x180;640x360;1280x720;1920x1080;
boxes_multisample=1x;2x;4x;

[Frame 0]
State=None
