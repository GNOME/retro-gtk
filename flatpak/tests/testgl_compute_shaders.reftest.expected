/testgl_compute_shaders/Boot: OK
/testgl_compute_shaders/Options: OK
/testgl_compute_shaders/0/State None: OK
/testgl_compute_shaders/0/Run: 
